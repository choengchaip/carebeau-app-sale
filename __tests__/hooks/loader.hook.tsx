import {useLoader} from "../../src/hooks/loader.hook";
import renderer from 'react-test-renderer';
import React, {useEffect, useState} from "react";

describe('Hook', () => {
  test('Loader', () => {
    const Mock = () => {
      const loader = useLoader({
        method: 'post',
        getURL: () => 'https://carebeauplus.amn-corporation.com/login/token',
      })

      return (
        <></>
      )
    }

    renderer.create(<Mock/>).toTree()
  })
});