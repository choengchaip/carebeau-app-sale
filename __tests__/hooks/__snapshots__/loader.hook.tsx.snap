// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hook Loader 1`] = `null`;
